$NetBSD: patch-Makefile,v 1.3 2017/09/02 18:15:42 tron Exp $

Don't attempt to build "keyanalyze" and related utilties. They don't build
on systems without SHA1 hash functions in the system library and we don't
install them in the first place.

--- Makefile.orig	2017-07-07 16:06:13.000000000 +0100
+++ Makefile	2017-09-02 19:14:30.000000000 +0100
@@ -1,6 +1,4 @@
--include /usr/share/dpkg/pkg-info.mk
-
-DIRS=caff gpg-key2ps gpg-mailkeys gpgsigs gpglist gpgparticipants keyanalyze keylookup \
+DIRS=caff gpg-key2ps gpg-mailkeys gpgsigs gpglist gpgparticipants keylookup \
      sig2dot springgraph gpgwrap gpgdir keyart gpg-key2latex
 TGZ=../signing-party_$(DEB_VERSION_UPSREAM).orig.tar.gz
 TGZ_DIR=signing-party-$(DEB_VERSION_UPSREAM)
